src/components/Resume/Resume.js

```diff
@@ -1,34 +1,50 @@
 import React, { forwardRef, useEffect, useRef, useState } from "react";
-import { Calendar } from "react-feather";
+import { Link } from "react-router-dom";
+import { useReactToPrint } from "react-to-print";
+import { Button, Radio } from "antd";
+import {
+  CalendarOutlined,
+  CheckSquareOutlined,
+  DownloadOutlined,
+  GithubOutlined,
+  LinkedinOutlined,
+  MailOutlined,
+  MobileOutlined
+} from "@ant-design/icons";
+import PropTypes from "prop-types";
 import joshImage from "../../assets/Josh-Logo-White-bg.svg";
-import { genderOptions, getMonthString } from "../../Constants";
-import jsonData from "./jsonData.json"
+import { genderOptions, getMonthString, PROFILES } from "../../Constants";
 import styles from "./Resume.module.css";
-//we cannt pass ref directly to component so we should wrap a component in forwardRef.
-const Resume = forwardRef((props, ref) => {
+
+const Resume = forwardRef(({ data }, ref) => {
+  const [profileType, setProfileType] = useState(PROFILES.internal);
+  Resume.propTypes = {
+    data: PropTypes.object.isRequired
+  };
   const {
-    showExperince = {},
-    showCertification = {},
-    information = {},
-    sections = {},
-    profile = {},
-    activeColor = {}
-  } = jsonData;
-  // const information = props.information;
-  // const sections = props.sections;
-  // const profile = props.profile;
-  const containerRef = useRef();
+    data: profile,
+    projectData: projects,
+    experienceData: experiences,
+    educationData: educations,
+    achievementData: achievements,
+    certificationData: certifications,
+    profiles
```

Comment:
do we need profiles here ? data: profile, should be sufficient

Reply:
yes , it is like internal or external profiles